index.html

```diff
@@ -82,6 +82,7 @@
         openmct.install(openmct.plugins.Filters(['table', 'telemetry.plot.overlay']));
         openmct.install(openmct.plugins.ObjectMigration());
         openmct.install(openmct.plugins.ClearData(['table', 'telemetry.plot.overlay', 'telemetry.plot.stacked']));
+        openmct.install(openmct.plugins.ConditionalStyles(['layout']));
```

Comment:
This could work, however any existing object types (ex. display layout) would need to be re-saved to have a styles property, right?
Otherwise only objects of that type that are created after our feature release will have the styles property.

Reply:
Good catch, this would require a migration script to be defined as well. Which is possible, but not desirable right now. 

There's a broader discussion to be had here about how plugins contribute styles but for right now let's get a prototype working without the need for defining a generic API.

Our current design has styling options for all create-able object types. A reasonable approach for now will be to check whether the object's type is a create-able type. You can do this using the Type API:

``` JavaScript
function isStyleInspectorVisible() {
    let type = openmct.types.get(object.type);
    return type.creatable;
}
```

Reply:
The next issue is how do we identify which style options apply to which objects. To do this properly requires some API. Let's learn what we need first with a hard-coded prototype, and then define some API if we need it.

Reply:
Done.